{"name": "test_login_01[100861]", "status": "broken", "statusDetails": {"message": "_pytest.fixtures.FixtureLookupError: ('data', <FixtureRequest for <Function test_login_01[100861]>>)", "trace": "file /Users/test/work/API_Testframe/TestCase/test_login.py, line 35\n      @allure.feature('Login')\n      @allure.severity('blocker')\n      @allure.story('测试登录成功')\n      @pytest.mark.parametrize(\"data\", list(Datas().data[1]['postdata'].values()) ,indirect=True)\n      def test_login_01(self, action,getdata,data):\nE       fixture 'data' not found\n>       available fixtures: action, cache, capfd, capfdbinary, caplog, capsys, capsysbinary, doctest_namespace, getdata, metadata, monkeypatch, pytestconfig, record_property, record_testsuite_property, record_xml_attribute, recwarn, tmp_path, tmp_path_factory, tmpdir, tmpdir_factory\n>       use 'pytest --fixtures [testpath]' for help on them.\n\n/Users/test/work/API_Testframe/TestCase/test_login.py:35"}, "description": "\n            用例描述：login接口\n        ", "attachments": [{"name": "log", "source": "4d73fc87-3199-408e-9e4a-f2e9a87d94f2-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "data", "value": "'10086'"}], "start": 1587626536331, "stop": 1587626536331, "uuid": "9f82ab10-a28c-437c-ba8b-f4a7ca37685c", "historyId": "2d66a24d28c7ef1d80e66726484d6a64", "testCaseId": "49d28fbbfb79bd9a490d1da4fd8fe957", "fullName": "TestCase.test_login.TestLogin#test_login_01", "labels": [{"name": "story", "value": "测试登录成功"}, {"name": "feature", "value": "Login"}, {"name": "severity", "value": "blocker"}, {"name": "parentSuite", "value": "TestCase"}, {"name": "suite", "value": "test_login"}, {"name": "subSuite", "value": "TestLogin"}, {"name": "host", "value": "Lettos-iMac.local"}, {"name": "thread", "value": "90990-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "TestCase.test_login"}]}